resources/css/bem/grid-items.less

```diff
@@ -2,26 +2,49 @@
 // See the LICENCE file in the repository root for full licence text.
 
 .grid-items {
-  --grid-items-gutter: 10px;
   display: flex;
   flex-wrap: wrap;
-  margin: calc(var(--grid-items-gutter) * -1);
-
-  // simulating `gap` property
-  > * {
-    margin: var(--grid-items-gutter);
-  }
+  margin: 0;
+  gap: 20px;
 
   &--2 {
-    --grid-items-gutter: 1px;
+    gap: 2px;
   }
 
   &--fade-out {
     opacity: 0.5;
   }
 
   &--notification-banner-buttons {
-    --grid-items-gutter: 5px;
+    gap: 10px;
     justify-content: center;
   }
+
+  &--ranking-filter {
+    gap: 10px 20px;
+    flex-direction: column;
+
+    @media @desktop {
+      align-items: center;
+      flex-direction: row;
+      justify-content: flex-end;
+    }
+  }
+
+  &--spotlights {
+    width: 100%;
+    gap: 0 20px;
+  }
+
+  &__item {
+    &--spotlight-user-filter {
+      width: 100%;
+      padding: 15px 0;
```

Comment:
This ends up making the gap between title and value on desktop quite large
![image](https://user-images.githubusercontent.com/1694544/226329329-248aa2c7-1b3c-4352-be4b-44d3fd1bb08f.png)

Can be removed and replaced with `align-self: center`